tests/src/test/scala/whisk/core/limits/ActionLimitsTests.scala

```diff
@@ -269,4 +269,29 @@ class ActionLimitsTests extends TestHelpers with WskTestHelpers {
             }
     }
 
+    it should "limit the length of HTTP req/resp bodies for debug messages to 1000 bytes" in withAssetCleaner(wskprops) {
+        (wp, assetHelper) =>
+            val auth: Seq[String] = Seq("--auth", wskprops.authKey)
+            val name = "largeFileAction"
+            val debugMsg = "Body excedes 1000 bytes and will be truncated"
+            val bodyMsg = "ENDUNIQUE"   //  Message that should be truncated
+
+            val actionCode = new File(s"$testActionsDir${File.separator}$name.js")
+            actionCode.createNewFile()
```

Comment:
Or if you really want to test create, you can pass a big parameter to any of the existing files in `action/data`.
```
val input = Map("payload" -> JsString("a" * 1000))
assetHelper.withCleaner(wsk.action, name) {
                (action, _) => action.create(name, file, parameters = input)
            }
```